### DISCUSSION TITLE
ext4 instead of btrfs

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#378] [Q&A]
on distrowatch, it says that spiral supports both btrfs and ext4 file systems. But when I install spiral, it defaults to btrfs. I want to install it using the ext4 file system; how do I do that ?

**Comment by geckolinux** [comment]
Hi there, it depends on the installation method. **Erase Disk** and **Manual Partitioning** will give you many more filesystem options.

**Reply by arky217** [reply]
I had already made the partition ext4, but when spiral installs, it changes it to btrfs.
And there seems to be no provision in the installation to install spiral as ext4.

**Reply by geckolinux** [reply]
It doesn't depend on the current filesystem of the partition, but rather on what filesystem you tell the Calamares installer to use. To do so you will need to use the **Erase Disk** or **Manual Partitioning** option and select EXT4.

**Reply by arky217** [reply]
I did use the manual partitioning option and selected a ext4 partition, but spiral changes it to btrfs when installing.

**Reply by geckolinux** [reply]
In that case you did something wrong. As mentioned, it doesn't matter what filesystem a partition _was using_, but rather what you set it _to be formatted_ to. It sounds like you're confusing the **Replace a partition** option with the **Manual Partitioning** option. Please post screenshots of what you're attempting in the **Manual Partitioning** section.

**Reply by arky217** [reply]
I finally figured out what I was doing wrong; thank you very much.

**Reply by geckolinux** [reply]
Glad you figured it out. Cheers.
